lib/ace/mode/behaviour/xml.js

```diff
@@ -116,7 +116,9 @@ var XmlBehaviour = function () {
                 if (position.column < tokenEndColumn)
                     return;
                 if (position.column == tokenEndColumn) {
-                    if (is(iterator.stepForward(), "attribute-value"))
+                    var nextToken = iterator.stepForward();
+                    // TODO also handle non-closed string at the end of the line
```

Comment:
I take it you'd like to release the current PR as a quick fix and then refine it?

Reply:
Yes, i think it requires changes to tokenizer